lib/notification-element.coffee

```diff
@@ -150,6 +150,22 @@ class NotificationElement extends HTMLElement
             #{packageCheck.latestVersion} latest.
             Upgrading to the latest version may fix this issue.
           """
+        else if packageCheck? and not packageCheck.upToDate and packageCheck.isCore
+          issueButton.remove()
+
+          fatalNotification.innerHTML += """
+            <br><br>
+            Locally installed core Atom package #{packageName} is out of date: #{packageCheck.installedVersion} installed locally;
```

Comment:
> Maybe `<code>` around the package name?

:+1: added in https://github.com/atom/notifications/commit/662ce73c62b3cb5d2d3042d21656890f5b969d85.